docs/docs/adrs/adr-010-standalone-changeover.md

```diff
@@ -0,0 +1,55 @@
+
+Next, the standalone consumer chain runs an upgrade which adds the CCV module, and is properly setup to execute changeover logic.
+
+The meat of the changeover logic is that the consumer chain validator set is updated to that which was specified by the provider via the queried consumer genesis. Validators which were a part of the old set, but not the new set, are given zero voting power. Once these validator updates are given to Comet, the set is committed, and in effect 2 blocks later (see [FirstConsumerHeight](../../../x/ccv/consumer/keeper/changeover.go#L19)).
+
+A relayer then establishes the new IBC connection between the provider and consumer. The CCV channel handshake is started on top of this connection. Once the CCV channel is established and VSC packets are being relayed, the consumer chain is secured by the provider.
+
+### Changes to CCV Protocol
+
+* Consumer Genesis state is updated to include a `PreCCV` boolean. When this boolean is set true in the  consumer genesis JSON, [special logic](../../../x/ccv/consumer/keeper/changeover.go) is executed on InitGenesis to trigger the changeover process on the consumer's first endblocker after the upgrade which adds the CCV module. Note that InitGenesis is not automatically called during chain upgrades, so the consumer must manually call the consumer's InitGenesis method in an upgrade handler.
+* The `ConsumerAdditionProposal` type is updated to include a `DistributionTransmissionChannel` field. This field allows the consumer to use an existing IBC transfer channel to send rewards as a part of the CCV protocol. Consumers that're not changing over from a standalone chain will leave this field blank, indicating that a new transfer channel should be created on top of the same connection as the CCV channel.
+* The CCV consumer keeper is updated to contain an optional reference to the standalone staking keeper. The standalone staking keeper is used to slash for infractions that happened before the changeover was completed. Ie. any infraction from a block height before the changeover, that is submitted after the changeover, will call the standalone staking keeper's slash method. Note that a changeover consumer's  standalone staking keeper becomes a democracy module keeper, so it is possible for a governance token to be slashed.
+
+## Consequences
+
+### Positive
+
+* Existing cosmos chains are now able to onboard over to a consumer chain secured by a provider.
+* The previous staking keepers for such chains can be transitioned to democracy staking module keepers.
+
+### Negative
+
+* The delineation between different types of consumers in this repo becomes less clear. Ie. there is code in the [democracy consumer's app.go](../../../app/consumer-democracy/app.go) that only applies to a previously standalone chain, but that file also serves as the base for a normal democracy consumer launched with RS from genesis.
```

Comment:
Use a github link instead of a relative link to `app.go`

Reply:
https://github.com/cosmos/interchain-security/commit/d5076870c16166a522fa538ec72a4502661c980f